packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -2635,6 +2635,40 @@ describe('<Autocomplete />', () => {
     });
   });
 
+  describe('prop: onScrollToBottom', () => {
+    it('should call onScrollToBottom when scroll reaches bottom', function test() {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        this.skip();
+      }
+      const onScrollToBottom = spy();
+      const { getByRole } = render(
+        <Autocomplete
+          open
+          options={['one', 'two', 'three', 'four', 'five']}
+          ListboxProps={{ style: { height: '100px' } }}
+          renderInput={(params) => <TextField {...params} />}
+          onScrollToBottom={onScrollToBottom}
+        />,
+      );
+      const textbox = getByRole('combobox');
+      act(() => {
+        textbox.focus();
+      });
+      const listbox = getByRole('listbox');
+      const option = screen.getAllByRole('option')[0];
+      const { height: optionHeight } = option.getBoundingClientRect();
+
+      const listboxPaddingBottom = Number(
+        window.getComputedStyle(listbox).getPropertyPriority('padding-bottom').replace('px', ''),
+      );
+
+      fireEvent.scroll(listbox, {
+        target: { scrollTop: optionHeight * 2 + listboxPaddingBottom },
```

Comment:
As per the video you posted in the description shouldn't this be 3 for three options?
```suggestion
        target: { scrollTop: optionHeight * 3 + listboxPaddingBottom },
```

Reply:
When 5th option is navigated, only 1st two options are hidden. So scrollTop value becomes (number of options hidden under scroll*height of each option) + padding